app/javascript/gobierto_dashboards/modules/costs_controller.js

```diff
@@ -105,7 +105,11 @@ export class CostsController {
 
     //Convert strings with some format to Numbers without format
     function convertStringToNumbers(amount) {
-      return Number(parseFloat(amount.replace(/\./g,'').replace(',','.')))
+      if (amount === '' || undefined) {
```

Comment:
`undefined` is being never evaluated. Upside down it does.
```suggestion
      if (amount === undefined || '') {
```